2024-RIKEN-AWS/JupyterNotebook/docker/Dockerfile.spawn

```diff
@@ -9,32 +9,51 @@ USER root
 ENV NB_USER=jovyan \
     NB_UID=1000 \
     HOME=/home/jovyan
+    # VENV_DIR=/home/jovyan/.flux_tutorial_venv
```

Comment:
Same here - can this commented out line be removed?

Reply:
I've got a commit that will remove that